examples/components/CCFRAUD/traininsilo/run.py

```diff
@@ -162,10 +163,18 @@ def __init__(
             self.train_sampler_ = None
             self.test_sampler_ = None
 
+        # get number of cpu to load data for each gpu
+        num_workers_per_gpu = int(
+            multiprocessing.cpu_count() // int(os.environ["WORLD_SIZE"])
```

Comment:
let's use `os.environ.get("WORLD_SIZE", "1")` instead of os.environ["WORLD_SIZE"] to avoid any errors when running locally.